lib/ae_mdw/db/name.ex

```diff
@@ -354,6 +359,17 @@ defmodule AeMdw.Db.Name do
   #
   # Private functions
   #
+  defp do_list_inactivated_at(plain_names, height, prev_plain_name) do
+    # includes revoked
+    case Database.next_key(Model.InactiveNameExpiration, {height, prev_plain_name}) do
+      {:ok, {^height, plain_name}} ->
+        do_list_inactivated_at([plain_name | plain_names], height, plain_name)
+
+      _other_height ->
+        plain_names
+    end
```

Comment:
This can be done using
```elixir
Collection.stream(Model.InactiveNameExpiration, :forward, {{height, Util.min_int()}, {height, Util.max_int()}}, nil)
```

Reply:
no problem, we can adopt the Collection lazy stream instead of iterator for any key iteration